lib/src/widgets/chat/video/ds_video_player.widget.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:flutter/material.dart';
+
+import 'package:chewie/chewie.dart';
+import 'package:get/get.dart';
+
+import 'package:blip_ds/blip_ds.dart';
+import 'package:blip_ds/src/controllers/ds_video_player.controller.dart';
+
+class DSVideoPlayer extends StatelessWidget {
+  final DSVideoPlayerController controller;
+  final String appBarText;
+
+  /// Video player widget
+
+  /// In this video player, the slash text is passed by the [appBarText] parameter so that it is contractually shown in the slash.
+  /// The [url] parameter contains the reference to be executed
+  DSVideoPlayer({
+    Key? key,
+    required this.appBarText,
+    required String url,
+  })  : controller = Get.put(DSVideoPlayerController(url: url)),
+        super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: Colors.black,
+      padding: EdgeInsets.only(top: Get.mediaQuery.padding.top),
+      child: Obx(
+        () {
+          return Scaffold(
+            extendBodyBehindAppBar: true,
+            backgroundColor: Colors.transparent,
+            body: Stack(
+              children: [
+                Column(
+                  children: <Widget>[
```

Comment:
This type annotation isn't recommended anymore.

```suggestion
                  children: [
```

Reply:
Done